Week1/homework/js-exercises/typeChecker.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+let x="fish";
+let y="cat";
+let person1={ "name": "Mr. Brown", "age":45};
+let person2={"name": "Mrs. Brown", "age":40};
+
+console.log(x);
+console.log(y);
+console.log(person1);
+console.log(person2);
+
+if (typeof x === typeof y){
+console.log("Same Type");
+} else {console.log ("Not the Same Type");
+}
+
+if (typeof x === typeof person1){
+console.log("Same Type");
+} else {console.log ("Not the Same Type");
+}
+
+if (typeof y === typeof person2){
```

Comment:
If you write a if statement it's often considered best practice to split up the lines as follows.

if(check) {
   // do something
} else {
   // do something else
}